evc-sumo/docker/install.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+#  Copyright (C) 2023 LEIDOS.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+set -e
+#user must replace the content in docker/token.txt with their own github token
+file="docker/token.txt"
```

Comment:
I'd recommend taking this in as an environment variable rather than a file. That way you can have the dockerfile take in the token as an `ARG ...` instruction, set it as an environment variable for the script as an `ENV ...` instruction and then have this script reference the token just as an environment variable. This way there's less risk of someone adding their token to the repo accidentally when committing.

Reply:
If an example of that would be helpful, I'd recommend checking out the ssc-interface-wrapper repo's dockerfile. It's not 100% the same, but that one manages a private license key in a similar way to what I've described.

Reply:
updated in the latest version, now need to input token during build